includes/class-wcpdf-setup-wizard.php

```diff
@@ -246,6 +246,7 @@ public function save_step() {
 						} else {
 							$settings[$key] = call_user_func( $sanitize_function, $value );
 						}
+						$settings[$key] = wp_kses( array_map( 'stripslashes', $value ), array() );
```

Comment:
by doing this, you are bypassing/overriding all this code above it in the loop:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/3bac479307ae3ebae0b491c28598ba9e52955307/includes/class-wcpdf-setup-wizard.php#L238-L248
What you should do instead, is applying this to the `$value` at the start of the loop.

`wp_kses` does not need to be applied because we're already sanitizing with the above functions. 

Finally, you can use `stripslashes_deep` in place of `array_map( 'stripslashes', $value )`, which is better because it can also handle `string` variables.

Reply:
i had strip slashes the `$value` at the bottom because i thought doing it at the top would be over written at the end of the loop. i'll add the update